include/ftl/fiber.h

```diff
@@ -56,45 +55,41 @@ inline std::size_t RoundUp(std::size_t numToRound, std::size_t multiple);
 
 typedef void (*FiberStartRoutine)(void *arg);
 
-
 class Fiber {
 public:
 	/**
 	 * Default constructor
 	 * Nothing is allocated. This can be used as a thread fiber.
 	 */
-	Fiber()
```

Comment:
I would prefer initializer arguments to be on newlines if possible. I'm not sure if clang format can do that

Reply:
As noted in the notes, the setting only dictates if it will go to one per line after it can no longer fit on a single line. You can't just tell it to always do one per line.

Reply:
I can live with it. Clang format is nice to have

Reply:
You can fake a lot of things with carefully placed comments which force clang-format's hand. If you look at my codebase I have stuff all over the place with strangely placed comments which force a line break. The good news is it always seems to get it right after you do force it's hand.